src/components/ArticleStrip/ArticleStrip.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './ArticleStrip.scss';
+import { ResponsiveImage, TextGroup } from '..';
+import { stylizedTimestamp } from '@quartz/js-utils';
+
+const responsiveImagePropsMapping = {
+	small: {
+		fallbackWidth: 80,
+		fallbackHeight: 80,
+		sizes: '80px',
+		widthRange: [ 80, 80 ],
+	},
+	'extra-large': {
+		fallbackWidth: 220,
+		fallbackHeight: 220,
+		sizes: `
+			(min-width: 1200px) 220px,
+			(min-width: 768px) 175px,
+			80px
+		`,
+		widthRange: [ 80, 220 ],
+	},
+};
+
+function ArticleStrip ( {
+	dateGmt,
+	edition,
+	kicker,
+	size,
+	thumbnailUrl,
+	title,
+} ) {
+	return (
+		<div className={`${styles.container} ${styles[ size ]}`}>
+			<div className={`${styles.thumbnailContainer} ${styles[ size ]}`}>
+				<ResponsiveImage
+					alt=""
+					src={thumbnailUrl}
+					{...responsiveImagePropsMapping[ size ]}
+				/>
+			</div>
+			<div>
+				<TextGroup
+					isArticle={true}
+					kicker={kicker}
+					size={size}
+					title={title}
+					tagline={`${stylizedTimestamp( dateGmt )} • ${edition}`}
+				/>
+			</div>
+		</div>
+	);
+}
+
+ArticleStrip.propTypes = {
+	/**
+	 * A string-based representation of the article publish datetime at
+	 * the Greenwich Mean Time (GMT-00:00:00). E.g.,
+	 * `'2012-09-04T17:02:10'`.
+	*/
+	dateGmt: PropTypes.string.isRequired,
+
+	/**
+	 * The Quartz edition to which the article belongs.
+	 */
+	edition: PropTypes.oneOf( [
+		'Quartz',
+		'Quartz Africa',
+		'Quartz India',
+		'Quartz at Work',
```

Comment:
have you worked through the move from slug to name? not 100% that's available everywhere we might want to use this component

Reply:
Can you think of any examples? As far as I can tell, we can map slugs to names anywhere in qz-react using this config file https://github.com/Quartz/qz-react/blob/master/common/config/meta/index.js